e2e/ios/E2EOktaReactNative/Info.plist

```diff
@@ -46,7 +46,7 @@
 	<string>SplashScreen</string>
 	<key>UIRequiredDeviceCapabilities</key>
 	<array>
-		<string>armv7</string>
+		<string>armv64</string>
 	</array>
```

Comment:
This was suggested by react-native upgrade helper, but I didn't see any practical differences from doing this. Still a good practice to stick with the mainstream